DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/DetailedProgress.cshtml

```diff
@@ -0,0 +1,106 @@
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateProgress
+@model DetailedCourseProgressViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegateProgress.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Detailed progress";
+  var routeParamsForBackLink = new Dictionary<string, string?> {
+    { "progressId", Model.ProgressId.ToString() },
+    { "accessedVia", Model.AccessedVia.ToString() },
+  };
+}
+
+@section NavBreadcrumbs {
+  @if (Model.AccessedVia.Equals(DelegateProgressAccessRoute.CourseDelegates)) {
+    <partial name="~/Views/TrackingSystem/Delegates/Shared/_CourseDelegatesProgressBreadcrumbs.cshtml" model="(Model.CustomisationId, Model.ProgressId)" />
+  } else {
+    <partial name="~/Views/TrackingSystem/Delegates/Shared/_ViewDelegateProgressBreadcrumbs.cshtml" model="(Model.DelegateId, Model.ProgressId)" />
+  }
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <div class="nhsuk-grid-column-two-thirds">
+      <h1 class="nhsuk-heading-xl">@ViewData["Title"]</h1>
+    </div>
+    <div class="nhsuk-grid-column-one-third heading-button-group">
+      <a class="nhsuk-button heading-button" role="button" href="@Model.ProgressDownloadUrl">
+        Download
+      </a>
+    </div>
+    <div class="nhsuk-card">
+      <div class="nhsuk-card__content basic-card-content">
+        <dl class="nhsuk-summary-list">
+          <div class="nhsuk-summary-list__row basic-summary-list__row">
+            <dt class="nhsuk-summary-list__key">
+              Name
+            </dt>
+            <dd class="nhsuk-summary-list__value">
+              @Model.DelegateName
+            </dd>
+          </div>
+          <div class="nhsuk-summary-list__row basic-summary-list__row">
+            <dt class="nhsuk-summary-list__key">
+              Email
+            </dt>
+            <partial name="_SummaryFieldValue" model="@Model.DelegateEmail" />
+          </div>
+          <div class="nhsuk-summary-list__row basic-summary-list__row">
+            <dt class="nhsuk-summary-list__key">
+              Delegate ID
+            </dt>
+            <dd class="nhsuk-summary-list__value">
+              @Model.DelegateNumber
+            </dd>
+          </div>
+          <div class="nhsuk-summary-list__row basic-summary-list__row">
+            <dt class="nhsuk-summary-list__key">
+              Last access
+            </dt>
+            <dd class="nhsuk-summary-list__value">
+              @Model.LastUpdated

```

Comment:
Just noticed this in the Show and Tell, but the dates appeared to be displayed as both dates and times, everywhere else we just display the dates. This should probably be made consistent so we only display the date here.

Reply:
I am going to be annoying and use the exact same logic to argue that the other delegate progress pages were shown off this morning as part of 649, and no complaints were made at the lack of times. My interpretation is that we only show dates as a general rule. The exception to that rule is the learning log, as that is specifically a _log_ consisting of a number of entries pertaining to the same subject.
Probably best to ask about these pages in hee-dls-refactor.

Reply:
Fair point. @JBsoftwire would you do the honours?

Reply:
Looks like the answer is that we should indeed display the times here. So my previously recommended change will need to be undone, apologies.